### PR TITLE
[converter/core] feature: Prepare Camunda 8.2

### LINKED ISSUE #166: Task: Prepare conversion rules for 8.2 release
### What should we do?

Currently, not all BPMN elements can be converted, and each bpmn element has a version in which it will be available.

This needs to be carried out.

Helpful resources are:
- Camunda Blog: https://camunda.com/blog/
- The Zeebe release page: https://github.com/camunda/zeebe/releases

### Why should we do it?

The converter needs to be up-to-date

### PR DESCRIPTION
**Pull request by jonathanlukas**
## Description
As Camunda 8.2 will cover more BPMN elements, we need to adjust our ruleset to fit the progress.

This PR will cover the following changes:

- [x] Support Task
- [x] Support Escalation
- [x] Support Error Codes as expressions
- [x] Support Link Events
- [x] Support embedded feel scripts in Script Tasks
- [x] Support Signal Start Events

## Additional context
Closes #166 

## Testing your changes
Every change is tested in 2 ways: Old version behaviour, new version behaviour.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an existing open issue)
- [x] New feature (non-breaking change which adds functionality to an extension)
- [ ] Breaking change (fix or feature that would cause existing functionality of an extension to change)
- [ ] Documentation update (changes made to an existing piece of documentation)

## Checklist:
- [x] My code adheres to the syntax used by this extension.
- [x] My pull request requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **Camunda Community Hub** documentation.
- [x] I have read the **Pull Request Process** documentation.
- [x] I have added or suggested tests to cover my changes suggested in this pull request.
- [x] All new and existing CI/CD tests passed.
- [x] I will /assign myself this issue, and add the relevant [issue labels] to it if they are not automatically generated by Probot.
- [x] I will tag @camunda-community-hub/devrel in a new comment on this issue if 30 days have passed since my pull request was opened and I have not received a response from the extension's maintainer.

### CODE REVIEW

**Comment by jonathanlukas** [comment]
@ingorichtsmeier @shaarmann It's time for a review!

File: backend-diagram-converter/core/src/main/resources/message-templates.properties [hunk 1]
```diff
@@ -114,6 +127,9 @@ delegate-implementation.severity=REVIEW
 script-job-type.message=Element '{{ elementLocalName }}' was transformed. Currently, script tasks are implemented like service tasks with job type '{{ jobType }}'. Please review your implementation.
 script-job-type.severity=REVIEW
 #
+result-variable-internal-script.message=Result variable is set to Zeebe script result variable. Please review.
+result-variable-internal-script.severity=REVIEW
```
Comment (ingorichtsmeier) [inline backend-diagram-converter/core/src/main/resources/message-templates.properties:131]: Could be INFO, as the variable name didn't change and is a fixed value.

File: backend-diagram-converter/core/src/main/resources/message-templates.properties [hunk 2]
```diff
@@ -187,10 +203,20 @@ inclusive-gateway-join.severity=WARNING
...
 script-format.message=Script format '{{ scriptFormat }}' was set to header '{{ headerName }}'. Please review.
 script-format.severity=REVIEW
 #
 script-format-missing.message=Script format could not be found. Please review.
-script-format-missing.severity=TASK
\ No newline at end of file
+script-format-missing.severity=TASK
+#
+escalation-code.message=Escalation code is transformed from '{{ oldCode }}' to '{{ newCode }}'. Please review.
+escalation-code.severity=REVIEW
```
Comment (ingorichtsmeier) [inline backend-diagram-converter/core/src/main/resources/message-templates.properties:218]: Is this a value or an expression? If it is a value, INFO would be OK.

File: backend-diagram-converter/core/src/main/resources/message-templates.properties [hunk 3]
```diff
@@ -187,10 +203,20 @@ inclusive-gateway-join.severity=WARNING
...
+#
+error-code.message=Error code is transformed from '{{ oldCode }}' to '{{ newCode }}'. Please review.
+error-code.severity=REVIEW
```
Comment (ingorichtsmeier) [inline backend-diagram-converter/core/src/main/resources/message-templates.properties:221]: See my comments above

File: backend-diagram-converter/core/src/main/resources/message-templates.properties (hunk 2 above)
Comment (jonathanlukas) [reply backend-diagram-converter/core/src/main/resources/message-templates.properties:218]: Escalation code is an expression 👍

File: backend-diagram-converter/core/src/main/resources/message-templates.properties (hunk 1 above)
Comment (jonathanlukas) [reply backend-diagram-converter/core/src/main/resources/message-templates.properties:131]: Fixed

File: backend-diagram-converter/core/src/main/resources/message-templates.properties (hunk 3 above)
Comment (jonathanlukas) [reply backend-diagram-converter/core/src/main/resources/message-templates.properties:221]: Same here. As long as this does not become an expression, there is no action to be taken and this message is irrelevant.

**Pull request merged**